src/main/java/edu/ucar/unidata/rosetta/controller/TemplateController.java

```diff
@@ -260,6 +480,7 @@ public String getBlankLines(@RequestParam("fileName") String fileName, @RequestP
      * @param request The HttpServletRequest with which to glean the client IP address.
      * @return A String of the local file name for the ASCII file (or null for an error).
      */
+    /*
     @RequestMapping(value = "/autoConvertKnownFile", method = RequestMethod.POST)
     @ResponseBody
     public String autoConvertKnownFile(UploadedAutoconvertFile file, HttpServletRequest request) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)